docs/source/_templates/theme_variables.jinja

```diff
@@ -14,5 +14,7 @@
   'blog': 'https://www.pytorchlightning.ai/blog',
   'resources': 'https://lightning-flash.readthedocs.io/',
   'support': 'https://lightning-flash.readthedocs.io/',
+  'community': 'https://pytorch-lightning.slack.com',
+  'forums': 'https://pytorch-lightning.slack.com',
```

Comment:
Maybe the invitation link here? New people cannot get to the workspace with the current link.
```suggestion
  'community': 'https://pytorch-lightning.slack.com/join/shared_invite/zt-pw5v393p-qRaDgEk24~EjiZNBpSQFgQ',
  'forums': 'https://pytorch-lightning.slack.com/join/shared_invite/zt-pw5v393p-qRaDgEk24~EjiZNBpSQFgQ',
```

Reply:
@akihironitta Yeah, although I can't see where this link actually appears in the docs, so maybe doesn't matter haha